app/src/main/java/org/oppia/android/app/application/alphakenya/AlphaKenyaApplicationComponent.kt

```diff
@@ -67,7 +67,7 @@ import javax.inject.Singleton
 @Singleton
 @Component(
   modules = [
-    ApplicationModule::class, DispatcherModule::class, LoggerModule::class, OppiaClockModule::class,
```

Comment:
Why is ApplicationModule being removed?

Reply:
Accidentally  removed this; adding it back